doc/usermanual/lighttable/concepts/filtering.xml

```diff
@@ -56,8 +56,10 @@
     Images in the lighttable view can be displayed in different sort orders, depending on
     <quote>filename</quote>, <quote>time</quote> (when the photo was taken),
     <quote>rating</quote> (i.e. stars), <quote>id</quote> (an internal ordering number of
-    darktable), or <quote>color labels</quote>. You can sort in reversed order by toggling the
-    triangle button to the right of the <quote>sort by</quote> combobox.
+    darktable), <quote>color labels</quote>, <quote>groups</quote>, <quote>full path</quote>,
+    <quote>title</quote>, <quote>description</quote>, <quote>aspect ratio</quote> or
+    <quote>custom order</quote> (user can drag &amp; drop images). You can sort in reversed order
```

Comment:
Playing a bit around with this feature it looks to me that you can only change custom order if the lighttable view is in file manager mode. When in zoomable lighttable mode you only pan the whole set of images.

Reply:
Right. Will update the doc.